src/any_llm/providers/gemini/utils.py

```
@@ -45,6 +45,23 @@ def _has_json_schema_refs(schema: Any) -> bool:
     return False
 
 
+def _strip_additional_properties(schema: Any) -> Any:
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `schema`

(ANN401)